x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/waterfall_flyout.test.tsx

```diff
@@ -13,10 +14,10 @@ import {
   REQUEST_HEADERS,
   RESPONSE_HEADERS,
 } from './waterfall_flyout';
-import { WaterfallMetaDataEntry } from '../types';
+import { WaterfallMetadataEntry } from '../../waterfall/types';
 
 describe('WaterfallFlyout', () => {
-  const flyoutData: WaterfallMetaDataEntry = {
+  const flyoutData: WaterfallMetadataEntry = {
```

Comment:
Thanks for changing the metadata naming, we could've held off on it but it's nice to be cleared up 🙂